src/client/containers/Admin/Routing/DeliveryConfigTemplateEdit.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Components } from '@opuscapita/service-base-ui';
+import translations from '../../i18n'
+
+export default class DeliveryConfigTemplateEdit extends Components.ContextComponent {
+
+    constructor(props, context) {
+        super(props);
+
+        context.i18n.register('RoutingAdmin.DeliveryConfigTemplate.Edit', translations);
```

Comment:
I will move this to the `componentWillMount()` and `componentDidMount()` methods, cause this will not work with language changes

Reply:
It should be in both methods or in one of them?

Reply:
In both methods. So something like 
```suggestion
       componentWillMount(){
           this.context.i18n.register('RoutingAdmin.DeliveryConfigTemplate.Edit', translations);
      }
      
      componentWillReceiveProps(nextProps, nextContext) {
          nextContext.i18n.register('RoutingAdmin.DeliveryConfigTemplate.Edit', translations);
    }
```